### ISSUE TITLE
Video is not playing

### ISSUE THREAD

**Issue by kpnkarthik** [stemkoski/AR-Examples#1]
The video which you have inside the videos are not playing with hiro pattern. Please help me how to fix this?

**Comment by kylejcorrigan** [comment]
hi, I'm having an issue, that is similar to this.  Mine is that the video example works fine on my laptop, but on my phone(iphone X) it doesn't load up the video.

**Comment by Bruce0418** [comment]
I have the same issue, the video is stuck with the black screen when the hiro maker is recognized. I will solve this issue and publish the solution when I have spare time. thanks

**Comment by matorlinc** [comment]
> I have the same issue, the video is stuck with the black screen when the hiro maker is recognized. I will solve this issue and publish the solution when I have spare time. thanks

Any progress? got the same issue, too

**Comment by matorlinc** [comment]
Guys, problem solved! Just comment the code below:(or replace with your own audio path)
`<!-- <source src="video/sintel.ogv" type='video/ogg; codecs="theora, vorbis"'> -->`

**Comment by jamess922** [comment]
@matorlinc  did as you say   but not working with my iphone 7

**Comment by fffiloni** [comment]
Hello people, i find a way to make the video starting on iOS:

Here are the steps:

1. Copy/paste this in the `head`:
```
<meta name="apple-mobile-web-app-capable" content="yes">
```

2. Just after your opening `body` tag, declare your video object:
```
<video id="video" autoplay loop muted playsinline crossorigin="anonymous" webkit-playsinline style="display:none">

<source src="videos/yourvideo.mp4 ">
</video>
```

3. At the very end of your script, you need to trigger a function which will play the video and tell iOS it’s okay:
```
function playVideo(){
	let video = document.getElementById('video');
	video.play();
}
playVideo();
```

It should work. It works for me. ;)

**Comment by jamess922** [comment]
@jamess922   video is working   but no sound?

**Comment by fffiloni** [comment]
Yeah its because in the video tag, i put the `muted` attribute. Remove it ;)

**Comment by jamess922** [comment]
> Yeah it’s because in the video tag, i put the `muted` attribute. Remove it ;)

here is problem  with muted  the video could play but no sound on my iphone 7  ,once i remove it  video not play on my iphone 7

**Comment by jamess922** [comment]
I googled this muted attribute,looks we need a button to maul play the video it's about Html5

**Comment by fffiloni** [comment]
Okay what you can do is leaving the muted attribute in the video tag.

Then in the last function, after `video.play` you can add : 
`video.muted = false;`

Tell me if it works

**Comment by jamess922** [comment]
Not working

**Comment by fffiloni** [comment]
Replace the `video.muted=false` from my previous answer by this:

```
var toggleSound = document.body.addEventListener('touchstart', function(){
        if(video.muted === true){
          video.muted=false;
        } else if(video.muted === false){
          video.muted=true;
        }
      });
```

It will toggle sound on touch event on tour screen. It works, just verified ;)

**Comment by jamess922** [comment]
@fffiloni it's working Thank you

**Comment by jamess922** [comment]
Hi @fffiloni   do you  know how to change the position of video ? I don't want the video on the center of marker ,and  I tried with `<a-video position="0 1 -1"></a-video>` Not working   and   `	renderer.domElement.style.position = 'absolute'
	renderer.domElement.style.top = '0px'
	renderer.domElement.style.left = '0px'`
this code could change the position but when i move my phone close to the marker the position will change......

**Comment by grebslam** [comment]
Can't get anything but a black box to display from the Video Texture example on: https://stemkoski.github.io/AR-Examples/

I tried adding the code posted here, and copied all github hosted assets to my own server to run locally, still nothing.

Even trying the example on github does not work, I'm on a Nexus 6 running 7.1.1 if that helps, tried both chrome and firefox.

Any further advice would be much appreciated.

**Comment by dannroda** [comment]
The issue is that is not allowed using autoplay with audio. It fixes by adding the `muted` property to the video tag
`<video id="video" muted autoplay...`

As the console says:
`Autoplay is only allowed when approved by the user, the site is activated by the user, or media is muted.`
